doc/source/user_guide/visualization.rst

```diff
@@ -1641,3 +1641,39 @@ when plotting a large number of points.
    :suppress:
 
     plt.close('all')
+
+Plotting backends
+-----------------
+
+Starting in 0.25 pandas can be extended with third-party plotting backends. The
+main idea is letting users select a plotting backend different than the provided
+one based on Matplotlib. For example:
+
+.. code-block:: python
+
+    >>> pd.set_option('plotting.backend', 'backend.module')
+    >>> pd.Series([1, 2, 3]).plot()
+
+Same thing can be done by setting the ``pd.options.plotting.backend`` keyword
+equals to 'backend.module'. For example:
+
+.. code-block:: python
+
+    >>> pd.options.plotting.backend = 'backend.module'
+    >>> pd.Series([1, 2, 3]).plot()
+
+This would be more or less equivalent to:
+
+.. code-block:: python
+
+    >>> import backend.module
+    >>> backend.module.plot(pd.Series([1, 2, 3]))
+
+The backend module can then use other visualization tools (Bokeh, Altair, hvplot,...)
+to generate the plots.
+
+Developers guide can be found at
+https://dev.pandas.io/docs/development/extending.html#plotting-backends
+
+More information on how to implement a third-party plotting backend can be found at
+https://github.com/pandas-dev/pandas/blob/master/pandas/plotting/__init__.py#L1.
```

Comment:
This sentence is not needed I think, since that will be included in the developer guide mentioned just above

Reply:
Done